### DISCUSSION TITLE
Customization for skins/Argon in Lazer.

### DISCUSSION THREAD

**Discussion by htmlpaws** [ppy/osu#20442] [Ideas] [answered]
My idea is that with skins, in particular the new "Argon" skin, the "player" (tho we all know osu's the one playing you) can go to the skin customization menu and modify additional elements of gameplay for the skin like combo colors, thickness of hit circles for Argon, cursor color modification, etc. along with the current UI changes that can be done with it. 

Maybe along with it, there can be a way to export the config, maybe if it's even just for Argon, and allow for the user to have different setups like if they were full skins, allowing for custom skins at a fraction of the file size, along with making the Default skin more viable for people who like messing with things to a level.

**Comment by peppy** [comment, marked as answer]
Please see the changelog. pull requests or youtube video where this is already touched on.

**Reply by htmlpaws** [reply]
Ah, forgot to check the log, oops!
